README.adoc

```diff
@@ -71,12 +72,12 @@ scripts such as this one. You need only run the following in a Terminal console:
 [source,console]
 ----
 $ bash <(curl -s https://raw.githubusercontent.com/metanorma/metanorma-macos-setup/master/metanorma-setup)
-$ gem install metanorma-csand
+$ gem install metanorma-csa
 
 ----
 
 == Documentation
 
-See https://www.metanorma.com/author/csand/[Author Cloud Security Alliance documents with Metanorma-CSAND].
+See https://www.metanorma.com/author/csa/[Author Cloud Security Alliance documents with Metanorma-CSAND].
```

Comment:
@ronaldtse will we rename this in future or I should revert this change?

Reply:
Yes let's rename it. (please also make a PR in metanorma.com)

Reply:
@ronaldtse https://github.com/metanorma/metanorma.com/pull/196